nx.json

```diff
@@ -198,6 +198,12 @@
       "outputs": ["{projectRoot}/dist", "{projectRoot}/build/.tsbuildinfo"],
       "cache": true
     },
+    "compile-github-action": {
+      "dependsOn": ["^build"],
+      "inputs": ["source", "^source"],
+      "outputs": ["{projectRoot}/packaged-action", "{projectRoot}/tmp"],
```

Comment:
is `"{projectRoot}/packaged-action"` really an output of the `build` script ? It seems to be only the output of the `package` script

Reply:
indeed